xml/ha_storage_protection.xml

```diff
@@ -578,25 +578,6 @@ Timeout (msgwait)  : 40
        <filename>/etc/sysconfig/sbd</filename> is added to the list of files
        that needs to be synchronized with &csync;.
       </para>
-      <para>
-       If you want to configure SBD manually, perform the following step:
-      </para>
-      <para>
-       To make the &corosync; init script start and stop SBD, edit the
```

Comment:
The paragraph is mainly about how to manually configure SBD AFAICT ;) Although we are not using init script any more, the configuration is the same. We can just delete the clause"To make the &corosync; init script start and stop SBD," in here I think.

Reply:
Thanks, I re-added how to configure SBD manually and just removed the clause as you suggested.